packages/palette/src/elements/Skeleton/Skeleton.story.tsx

```diff
@@ -0,0 +1,13 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+import { Skeleton } from "./Skeleton"
+
+storiesOf("Components/Skeleton", module).add("Basic", () => {
+  return (
+    <>
+      <Skeleton m={2} width={400} height={300} />
+      <Skeleton m={2} width={400} height={300} done />
+      <Skeleton m={2} width={400} height={300} borderRadius="1em" />
+    </>
+  )
+})
```

Comment:
Won't this always cause the visual tests to fail?

Reply:
Ah, good catch. I'll disable animation here for Chromatic.